modules/op-release-notes-1-11.adoc

```diff
@@ -235,3 +235,41 @@ With this update, {pipelines-title} General Availability (GA) 1.11.1 is availabl
 * Before this update, in {pipelines-title} 1.11.0, the Operator always installed {tekton-chains} and you could not disable installation of the {tekton-chains} component. With this update, you can set the value of the `disabled` parameter to `true` in the `TektonConfig` CR  to disable installation okindf {tekton-chains}.
 
 * Before this update, if you configured {tekton-chains} on an older version of {pipelines-shortname} using the `TektonChain` CR and then upgraded to {pipelines-shortname} version 1.11.0, the configuration information was overwritten. With this update, if you upgrade from an older version of {pipelines-shortname} and {tekton-chains} was configured in the same namespace where the `TektonConfig` is installed (`openshift-pipelines`), {tekton-chains} configuration information is preserved.
+
+[id="release-notes-1-11-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.2
+
+With this update, {pipelines-title} General Availability (GA) 1.11.2 is available on {OCP} 4.12 and later versions.
+
+This update includes an updated version of the `tkn` command line tool. You can download the updated version of this tool at the following locations:
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-s390x.tar.gz[Linux on {ibmzProductName} and {linuxoneProductName} (s390x)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-ppc64le.tar.gz[Linux on {ibmpowerProductName} (ppc64le)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-linux-arm64.tar.gz[Linux on ARM (aarch64, arm64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-windows-amd64.zip[Windows]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-macos-amd64.tar.gz[macOS]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/tkn-macos-arm64.tar.gz[macOS on ARM]
+
+If you installed the `tkn` command line tool using RPM on {op-system-base-full}, use the `yum update` command to install the updated version.
+
+[id="fixed-issues-1-11-2_{context}"]
+=== Fixed issues
+
+* Before this update, the `tkn pac resolve -f` command did not detect the existing secret for authentication with the Git repository. With this update, this command successfully detects the secret.
+
+* With this update, you can use `--v1beta1` flag in the `tkn pac resolve` command. Use this flag if the pipeline run is generated with the `v1beta1` API version schema.
+
+* Before this update, the `tkn pr logs` command failed to display the logs for a pipeline run if this pipeline run was created using a resolver. With this update, the command displays the logs.
```

Comment:
nit : Before this update, the `tkn pr logs` command failed to display the logs for a pipeline run if this pipeline run references a resolver. With this update, the command displays the logs.